tests/test_appliancesmanager.py

```diff
@@ -121,3 +121,100 @@ async def test_fetch_appliances_returns_true_if_either_method_returns_true(
 
     assert result == bool(owned_response or shared_response)
     await http_client_mock.close_session()
+
+
+@pytest.mark.parametrize(
+    "data_model_key",
+    [
+        "DDM_COOKING_BIO_SELF_CLEAN_TOURMALINE_V2",
+        "COOKING_MINERVA",
+        "COOKING_VSI",
+        "COOKING_U2",
+    ],
+)
+async def test_add_appliance(data_model_key, http_client_mock: AiohttpClientMocker):
+    appliance = {
+        "DATA_MODEL_KEY": f"{data_model_key}",
+        "CATEGORY_NAME": "Cooking",
```

Comment:
Can you move this data to the data/owned_appliances.json file?

Reply:
done